cve_bin_tool/sbom_manager/__init__.py

```diff
@@ -0,0 +1,132 @@
+# Copyright (C) 2021 Anthony Harrison
```

Comment:
I'm also curious if `Copyright (C) 2021 Anthony Harrison` is okay? Shouldn't it be `Copyright (C) 2021 Intel Corporation`?